var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -88,7 +90,10 @@ def cmake_args(self):
         options.append('-DBUILD_SHARED_LIBS={0}'.format(
             'On' if '+shared' in spec else 'Off'))
 
+        options.append('-DENABLE_BENCHMARKS={0}'.format(
+            'On' if '+benchmarks' in spec else 'Off'))
```

Comment:
I extended CMake packages a few months ago with [a helper function](https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html?highlight=define_from_variant#adding-flags-to-cmake) designed to simplify constructions like these:
```c++
options.append(self.define_from_variant('ENABLE_BENCHMARKS', 'benchmarks'))
```
but in the interests of uniformity with the surrounding text you can leave as-is.